app/configurations/config.hb.js

```diff
@@ -304,7 +304,7 @@ export default configMerger(walttiConfig, {
       {
         name: {
           fi: '',
-          en: 'Bicycle infrastructur',
+          en: 'Bicycle infrastructure',
```

Comment:
Good you spotted this. Could you fix this in another PR? We should keep this PR focused to the favorites, so others could profit from it as well. Otherwise LGTM

Reply:
Ok, will do.